app/ts/library/bundleUtils.ts

```diff
@@ -54,42 +48,38 @@ export const signBundle = async (bundle: FlashbotsBundleTransaction[], provider:
 			if (tx.transaction.from in inSimulation) accNonces[tx.transaction.from] -= inSimulation[tx.transaction.from]
 		}
 		tx.transaction.nonce = accNonces[tx.transaction.from]
-		console.log(tx.transaction.nonce)
 		const signedTx = await tx.signer.signTransaction(tx.transaction)
 		transactions.push(signedTx as string)
 	}
 	return transactions
 }
 
 export const createBundleTransactions = async (
-	interceptorPayload: BundleState | undefined,
+	bundle: Bundle | undefined,
 	signers: Signers,
 	blockInfo: BlockInfo,
 	blocksInFuture: bigint,
 	fundingAmountMin: bigint,
-	provider: providers.Web3Provider,
 ): Promise<FlashbotsBundleTransaction[]> => {
-	if (!interceptorPayload) return []
-	return await Promise.all(interceptorPayload.payload.map(async ({ from, to, nonce, gasLimit, value, input, chainId }, index) => {
+	if (!bundle) return []
```

Comment:
this could be handled outside the function and make this function only accept Bundle, then on the undefined case you would just set it []